src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,7 +14,9 @@
     private Phone phone;
     private Email email;
     private Address address;
-
+    private int sequenceNumber;
+    private static int nextSequenceNumber = 0;
```

Comment:
from a code readability point of view, it may make more sense to initialize it to 1 rather than 0.